app/posts/apply-for-teacher-training/2023-07-12-when-francis-felt-frustrated.md

```diff
@@ -0,0 +1,182 @@
+---
+title: When Franics felt frustrated
+description: We developed a story about subject knowledge enhancement courses. The purpose of the story is to show how we can empathise with users by capturing their pain points in an engaging way.
+date: 2023-07-12
+tags:
+  - subject knowledge enhancements
+ogImage:
+  src: francis-cover-image.png
+  alt: "Illustration showing a section called 'Interview availability in a task list. This screen then connects to the next screen with a question asking the user if there are any times they will not be available for interviews."
+---
+
+![Illustration showing a section called 'Interview availability in a task list. This screen then connects to the next screen with a question asking the user if there are any times they will not be available for interviews.](francis-cover-image.png)
+
+Previously, we designed guidance to tell candidates they have to do a [subject knowledge enhancement (SKE) course](/apply-for-teacher-training/telling-candidates-they-have-an-ske-condition/) before their teacher training.
+
+We tested these designs and learnt a lot about SKE courses. Some of the findings we uncovered were about the wider SKE policy and do not fit with Apply for teacher training (Apply).
```

Comment:
I wonder if this opening para could be a bit more punchy, and summarise (or at least hint at) the issues themselves? Might also be good to briefly summarise what the research consisted of?

This suggestion isn’t quite right either, but might be an approach to try?

```suggestion
We did some research around subject knowledge enhancement (SKE) courses, as part of [designing guidance telling candidates when they have to do these](/apply-for-teacher-training/telling-candidates-they-have-an-ske-condition/) before their teacher training.

We usability tested the designs with X participants, and also sent a survey to X people who had previously received an offer with a SKE course condition attached.

This research revealed some wider issues with how users choose a course, as well as the process of applying to a a course and receiving the funding, and the content of the course itself.
```